### ISSUE TITLE
OWL prefixes are lost

### ISSUE THREAD

**Issue by tillmo** [spechub/Hets#1858]
When analysing
```
logic OWL
ontology O =
Class: C
DataProperty: dateOf
              Domain: C
              Range: xsd:date
```
and then outputting the resulting theory (e.g. with `-o th`), the result has a `Range: date`, which leads to an error, because `date` is an unknown datatype.

**Comment by mcodescu** [comment]
We should test this again after the branch for iris is merged.

**Comment by mcodescu** [comment]
Solved on #1909
